pallets/moonbeam-xcm-benchmarks/src/mock.rs

```diff
@@ -69,7 +71,9 @@ parameter_types! {
 	pub WeightPrice: (AssetId, u128, u128) = (AssetId(Location::parent()), 1_000_000, 1024);
 }
 
+#[allow(dead_code)]
 pub struct AllAssetLocationsPass;
+
```

Comment:
Same, removing `pub` from `pub struct AllAssetLocationsPass;` should be enough.

Reply:
Also here, unfortunately it is not enough.